### PR TITLE
bigquerydatatransfer: convert to minitest

### PR DESCRIPTION
**Pull request by quartzmo**
The minitest test added in this PR passes locally:

```
quartzmo@Chriss-iMac bigquerydatatransfer % bundle exec rake test
Run options: --junit --junit-filename=sponge_log.xml --seed 38569

# Running:

.

Finished in 1.134512s, 0.8814 runs/s, 8.8144 assertions/s.

1 runs, 10 assertions, 0 failures, 0 errors, 0 skips
```

### CODE REVIEW

File: bigquerydatatransfer/quickstart.rb [hunk 1]
```diff
@@ -12,29 +12,32 @@
...
+  # [START bigquerydatatransfer_quickstart]
+  # [START require_library]
+  # Imports the Google Cloud client library
+  require "google/cloud/bigquery/data_transfer"
+  # [END require_library]
 
-# Your Google Cloud Platform project ID
-project_id = "YOUR_PROJECT_ID"
+  # Your Google Cloud Platform project ID
+  # project_id = "YOUR_PROJECT_ID"
```
Comment (quartzmo) [inline bigquerydatatransfer/quickstart.rb:23]: @TheRoyalTnetennba I wasn't sure what to do about this line in the sample. Just remove it, as you did in #542?

Comment (TheRoyalTnetennba) [reply bigquerydatatransfer/quickstart.rb:23]: yeah. might have to wrap the sample in a method so it's easier to test.

Comment (quartzmo) [reply bigquerydatatransfer/quickstart.rb:23]: I'm not sure I understand. What change should I make here?

Comment (quartzmo) [reply bigquerydatatransfer/quickstart.rb:23]: OK, based on similar [argument "initialization comments" in 542](https://github.com/GoogleCloudPlatform/ruby-docs-samples/blob/0b1fa9a0f3541eb32ec2a88b101db75d935c488c/logging/quickstart.rb#L27-L30), this should be good as-is. Thanks.

File: bigquerydatatransfer/quickstart.rb [hunk 2]
```diff
@@ -12,29 +12,32 @@
...
-data_transfer = BigQueryDataTransfer.new
+  # Instantiate a client
+  data_transfer = Google::Cloud::Bigquery::DataTransfer.new
 
-# Get the full path to your project.
-project_path = BigQueryDataTransfer::V1::DataTransferServiceClient.project_path(
-  project_id
-)
+  # Get the full path to your project.
+  project_path = Google::Cloud::Bigquery::DataTransfer::V1::DataTransferServiceClient.project_path(
```
Comment (TheRoyalTnetennba) [inline bigquerydatatransfer/quickstart.rb:29]: is there a path helper for this?

Comment (quartzmo) [reply bigquerydatatransfer/quickstart.rb:29]: I believe that is the correct path helper.

Comment (quartzmo) [reply bigquerydatatransfer/quickstart.rb:29]: @dazuma Does this path helper look right to you?

Comment (dazuma) [reply bigquerydatatransfer/quickstart.rb:29]: Yeah that looks correct.

**Pull request merged**